pkg/reconciler/workload/apiexportcreate/apiexportcreate_controller.go

```diff
@@ -171,19 +172,26 @@ func (c *controller) enqueue(obj interface{}) {
 		runtime.HandleError(err)
 		return
 	}
-	clusterName, name := clusters.SplitClusterAwareKey(clusterAwareName)
+	clusterName, _ := clusters.SplitClusterAwareKey(clusterAwareName)
 
-	klog.Infof("Enqueueing logical cluster %q because of %T %s", clusterName, obj, name)
-	c.queue.Add(clusterName.String())
+	key = clusterName.String()
+	logger := logging.WithQueueKey(logging.WithReconciler(klog.Background(), controllerName), key)
+	if logObj, ok := obj.(logging.Object); ok {
+		logger = logging.WithObject(logger, logObj)
+	}
+	logger.V(2).Info(fmt.Sprintf("queueing ClusterWorkspace because of %T", obj))
```

Comment:
why not structural?

Reply:
`WithValues("trigger", fmt.Sprintf("%T", obj))`

Reply:
You add `type.{workspace,namespace,name}` structured fields in the check above, so this print-formatter for the sentence is just explanatory - without it, what would you write? What if multiple objects are involved? You need to know the type that the trigger was to identify it (in our pattern, anyway).

Some more static "queueing Foo because of a trigger" and "trigger.{workspace,namespace.name}` fields could be fine but feels like another level of indirection for little gain

Reply:
k